src/MessagingBubble/MessagingBubble.less

```diff
@@ -15,26 +14,33 @@
 .MessagingBubble--Message--Own {
   background-color: @neutral_dark_gray;
   color: @neutral_white;
+  .padding--xs();
 }
 
 .MessagingBubble--Message--Other {
   background-color: #f3f5fd; /* stylelint-disable-line */
   border-radius: 0.3125rem;
   color: @neutral_black;
+  .padding--xs();
+}
+
+.MessagingBubble--Message--Reply--Parent {
+  .padding--m();
 }
 
 .MessagingBubble--Message--Reply--Own {
   background-color: #636776; /* stylelint-disable-line */
   border-radius: 0.3125rem;
   color: @neutral_white;
-  .margin--xs();
+  .margin--bottom--s();
 }
 
 .MessagingBubble--Message--Reply--Other {
   background-color: @neutral_off_white;
   border-radius: 0.3125rem;
   color: @neutral_black;
   .margin--xs();
+  .margin--left--none();
```

Comment:
Do the Own and Other have slightly different margin requirements?

Reply:
oops, typo